entry_types/scrolled/doc/creating_content_elements.md

```diff
@@ -155,7 +155,14 @@ Run the following command in the root directory of your host application
 [documentation below](#using-transcoded-files-in-storybook-or-percy)):
 
 ```bash
-$ bundle exec rake pageflow_scrolled:storybook:seed:setup[./seed.json]
+$ bundle exec rake pageflow_scrolled:storybook:seed:create_entry
+$ bundle exec rake pageflow_scrolled:storybook:seed:generate_json[./seed.json]
```

Comment:
Why can't we call `setup` here any longer? It should invoke the other two:

https://github.com/codevise/pageflow/blob/f51a288a5abe2df495ee644e80a64ff8b345a0b6/entry_types/scrolled/lib/tasks/pageflow_scrolled_tasks.rake#L9-L10

Reply:
I concluded that, since `bundle exec rake -T` from Dedicated only lists `create_entry`, `destroy_entry`, and `generate_json[output]`, `setup` wouldn't work anymore

Reply:
I just found out about `rake -T -A`, which shows `setup`, as well.

What do you think about the note on `zsh`? `zsh` has become more and more common, so maybe it makes sense to mention this kind of thing

Reply:
The setup task does not show up since it is missing a description. So maybe we should add that instead. I do not use zsh myself, but I'm totally fine with including the hint.